tests/_support/AcceptanceTester.php

```diff
@@ -256,6 +256,27 @@ public function iShouldSeeAnImageIsLinkedTo($image, $expectedUrl)
         $this->assertEquals($expectedUrl, $actualUrl);
     }
 
+    /**
+     * @When I click the table settings for :table
+     */
+    public function iClickTheTableSettings($table)
+    {
+        $this->click("#".$table);
+        $this->wait(1);
+    }
+
+    /**
+     * @Then I see checkbox :checkbox is checked
+     */
+    public function iSeeCheckboxIsChecked($checkbox)
+    {
+        $this->uncheckOption('#'.$checkbox);
+        $this->checkOption('#'.$checkbox);
```

Comment:
Having the above 2 statements seems strange to me based on the name of the function, `iSeeCheckboxIsChecked`. I would have created and used 2 new functions called `iUncheckCheckboxInTableSettings` and `iCheckCheckboxInTableSettings`.

Reply:
This function has been removed.